designs/virtualization/4.x/Templates/Templates.md

```diff
@@ -0,0 +1,57 @@
+# Templates
+
+## Virtual machines and VM templates are located under Virtualization
+
+Removing the tab ‘Virtual Machine Templates’ from the side navigation and combining it with ‘Virtual Machines’, so both are located under ‘Virtualization’.
+This PR will focus on how users can create a VM when they choose a to create a VM from a Template.
+
+### Virtual Machine Templates Tab
+
+From the Virtualization left nav item the user goes to the Virtual Machine Templates Tab where they can see the full list of templates.
+From this point, we suggest 2 options to create a VM:
+
+- Create VM from the selected templated via an action from the kebab menu
+
+![TemplatesTabOp1](img/TemplatesTabsOp1.png)
+
+- Create VM from the selected templated via an action from the kebab menu and also allow an action link
+
+![TemplatesTabOp2](img/TemplatesTabsOp2.png)
```

Comment:
Should the create action be a template?

Reply:
I think we can stick with "Create template" for the primary action here too just to keep it concise.